stable/stolon/templates/keeper-statefulset.yaml

```diff
@@ -128,11 +128,23 @@ spec:
           - name: config
             mountPath: /pre-stop-hook.sh
             subPath: pre-stop-hook.sh
+{{ end }}
+{{- if .Values.nodePostStartScript }}
+          - name: config
+            mountPath: /postStartScript.sh
+            subPath: postStartScript.sh
+{{ end }}
           lifecycle:
+{{- if .Values.keeper.hooks.failKeeper.enabled }}
             preStop:
               exec:
                 command: ["/bin/bash", "-e", "/pre-stop-hook.sh"]
 {{ end }}
+{{- if .Values.nodePostStartScript }}
+            postStart:
+              exec:
+                command: ["/bin/bash", "-e", "/postStartScript.sh"]
+{{ end }}
```

Comment:
Replace  `{{ end }}` to `{{- end}}`

Reply:
Should we reformat line number from 160 to 178 ?

Reply:
> Should we reformat line number from 160 to 178 ?

Seems ok. I haven't touched it

Reply:
> Replace `{{ end }}` to `{{- end}}`

Solved